source/community.rst

```diff
@@ -0,0 +1,16 @@
+At the time of writing, accessibility is not a regular consideration in many 
+Scientific Python projects. Many projects may have a handful of people who 
+have accessibility knowledge or are interested in accquiring more, but may not 
+be able to reach critical mass on a per-project basis. Connecting Scientific 
+Python community members focused on accessibility will provide a boon for 
+individuals, projects, and the ecosystem at large. 
+
+Working with the community may include activities that
+
+* Make the ecosystem more accessible 
+* Clarify the broad term accessibility and its variouspossible implementations
+* Provide specific assessment tools and steps
+* Direct to existing accessibility standards and modifications for open source 
+    specifically 
+* Provide case studies of how different core projects became more accessible
+* Start discussion and develop shared solutions
```

Comment:
Since I am directing people to github.com/scientific-python/accessibility.scientific-python.org, I can make a separate PR to update the README.md and a new issue template. These files would have the same information as here. Makes for a more consistent user journey across different websites.